components/collector/src/source_collectors/harbor_json/base.py

```diff
@@ -0,0 +1,30 @@
+"""Base classes for Harbor JSON collectors."""
+
+from typing import Final, TypedDict
+
+REPORT_MIME_TYPE: Final = "application/vnd.security.vulnerability.report; version=1.1"
+
+
+class Vulnerability(TypedDict):
```

Comment:
Perhaps a good moment to rename this to `HarborVulnerability` or `HarborJSONVulnerability` ?